cirq-aqt/cirq_aqt/aqt_sampler.py

```diff
@@ -25,23 +25,21 @@
 import json
 import time
 import uuid
-from typing import Iterable, List, Union, Tuple, Dict, cast, TYPE_CHECKING
+from typing import Iterable, List, Union, Tuple, Dict, cast
 
 import numpy as np
 from requests import put
-from cirq import circuits, Sampler, resolve_parameters, LineQubit
-from cirq.study.sweeps import Sweep
-from cirq.aqt.aqt_device import AQTSimulator, get_op_string
-from cirq import study, ops, IonDevice
 
-if TYPE_CHECKING:
-    import cirq
+import cirq
+from cirq_aqt.aqt_device import AQTSimulator, get_op_string
 
-Sweepable = Union[study.ParamResolver, Iterable[study.ParamResolver], Sweep, Iterable[Sweep]]
+Sweepable = Union[
+    cirq.ParamResolver, Iterable[cirq.ParamResolver], cirq.Sweep, Iterable[cirq.Sweep]
+]
```

Comment:
Can remove this and use `cirq.Sweepable` instead.

Reply:
Done (although this was not changed in this PR, but agree to clean it up!). In fact it was never used anywhere, so I just removed it.